microhapulator/data/marker_details_template.html

```diff
@@ -0,0 +1,183 @@
+            overflow-x: scroll;
+            font-family: 'Courier New', monospace;
+            font-size: 12pt;
+        }
+
+        ::-webkit-scrollbar {
+            width: 5px;
+            height: 5px;
+        }
+
+        ::-webkit-scrollbar-thumb {
+            background: gray;
+            border-radius: 10px;
+        }
+
+        ::-webkit-scrollbar-track {
+            -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+            border-radius: 10px;
+            background-color: #F5F5F5;
+        }
+    </style>
+</head>
+
+<body>
+
+
+    <div class="container" id="marker_detail_container">
+        <select onchange="show_selected_details(this.value)" id="drop_down">
+            {% for markername in markernames %}
+            <option>{{markername}}</option>
+            {% endfor %}
+        </select>
+        <input id="searchbar" , type="search" onsearch="search_marker()" name="search" placeholder="Search markers..">
+        <p></p>
+        {% for markername in markernames %}
+        <div class="marker_details" id={{markername}} style="display:none">
+            <p>
+            <h4>Length</h4>{{marker_details_table.loc[markername,'Length']}} bp</p><br>
+            <p>
+            <h4>GC content</h4>{{marker_details_table.loc[markername,'GC']}}%</p><br>
+            <p>
+            <h4>Offsets</h4><span class="offsets">{{marker_details_table.loc[markername,'Offsets']}}</span></p><br>
+            <p>
+            <h4>Sequence (offsets in red) </h4>
+            <div class="sequence">{{marker_details_table.loc[markername,'Sequence']}}</div>
+            </p>
+            <br><br><br>
+            <table>
+                <h4>Read Mapping</h4>
+                <tr>
+                    <th>Sample</th>
+                    <th class="alnrt">Mapped Reads</th>
+                    <th class="alnrt">Mapped Rate</th>
+                    <th class="alnrt">Off Target Reads</th>
+                    <th class="alnrt">Off Target Rate</th>
+
+                </tr>
+                {% for sample, sample_data in mapping_rates.items()%}
+                <tr>
+                    <td>{{sample}}</td>
+                    <td class="alnrt">{{"{:,.0f}".format(sample_data.loc[markername,'ReadCount'])}}</td>
+                    <td class="alnrt">{{"{:.2f}".format(sample_data.loc[markername, 'ExpectedObservedRatio'])}}</td>
+                    <td class="alnrt">{{"{:,.0f}".format(sample_data.loc[markername, 'OffTargetReads'])}}</td>
+                    <td class="alnrt">{{"{:.2f}".format(sample_data.loc[markername, 'OffTargetRate']*100)}}%</td>
+                </tr>
+                {% endfor %}
+            </table>
+            <br><br><br>
+            <table>
+                <h4>Haplotype Calling</h4>
+                <tr>
+                    <th>Sample</th>
+                    <th class="alnrt">Typed Reads</th>
+                    <th class="alnrt">Total Reads</th>
+                    <th class="alnrt">Typing Rate</th>
```

Comment:
The contents of this table are a bit more self-explanatory, but we should probably include a caption or explanatory narrative text here as well for completeness' sake.

Reply:
Also added